insightly.py

```diff
@@ -912,7 +913,7 @@ def ownerinfo(self):
         """
         :return: dictionary of information about the account owner
         """
-        return {'name': self.owner_name, 'email': self.owner_email, 'id': self.owner_id}
```

Comment:
There may be cases where owner_id may be useful to have as part of the ownerinfo, could we please keep it in addition to contact_id?